examples/hosted-wallet-kt/wallet/src/main/kotlin/ExampleCancelFlow.kt

```diff
@@ -0,0 +1,93 @@
+import tbdex.sdk.messages.*
+import tbdex.sdk.web5.BearerDid
+
+private lateinit var webhook: ReplyToWebhook
+private var closedReceived = false
+
+fun runCancelFlow(
+    pfiDidUri: String,
+    verifiableCredential: String,
+    bearerDid: BearerDid,
+    replyToUrl: String
+) {
+    println("\n ~Running Cancel Flow~ \n")
+
+    webhook = ReplyToWebhook(
+        onQuoteReceived = { quote ->
+            println("3. Quote received...")
+            println("Quote received: ${quote.metadata.id}\n")
+            handleCancelFlowQuoteReceived(quote, pfiDidUri, bearerDid)
+        },
+        onOrderStatusReceived = { orderStatus ->
+            println("Received order status ${orderStatus.metadata.id} ${orderStatus.data.status}\n")
```

Comment:
maybe add a comment, like

```suggestion
            // this example cancel flow submits the cancel prior to order submission, and so
            // no order statuses are expected to be received
            println("Received order status ${orderStatus.metadata.id} ${orderStatus.data.status}\n")
```

or maybe make the `onOrderStatusReceived` an optional parameter to the `ReplyToWebhook` and then just don't send in a value here?